.github/CODEOWNERS

```diff
@@ -34,9 +34,12 @@
 /sdk/batch/ @dpwatrous @paterasMSFT @zfengms @deyaaeldeen
 
 # PRLabel: %Communication
-/sdk/communication/ @DominikMe @0rland0Wats0n @ankitarorabit @Azure/azure-sdk-communication-code-reviewers
 /sdk/communication/communication-identity/ @petrsvihlik @martinbarnas-ms
 /sdk/communication/communication-chat/ @LuChen-Microsoft
+/sdk/communication/communication-phone-numbers/ @miguhern
+/sdk/communication/communication-network-traversal/ @AriZavala2
+/sdk/communication/communication-sms/ @RoyHerrod
+/sdk/communication/communication-common/ @petrsvihlik
```

Comment:
```suggestion
/sdk/communication/communication-identity/ @Azure/acs-identity-sdk 
```

Reply:
Updated owners for identity in https://github.com/Azure/azure-sdk-for-js/pull/17530/commits/83f265ea3838c97174008fcbc91ebb1c0fd48b76 to be the team you mentioned 

You want to use the same team for `communication-common` package too?

Reply:
yes

Reply:
```suggestion
/sdk/communication/communication-common/ @Azure/acs-identity-sdk
```